website/versionedDocsBlacklist.json

```diff
@@ -44,5 +44,6 @@
   "headless-js-android",
   "signed-apk-android",
   "removing-default-permissions",
-  "optimizing-flatlist-configuration"
+  "optimizing-flatlist-configuration",
+  "onviewableitemschanged-in-flatlist"
```

Comment:
The page id in blacklist is incorrect.

Reply:
Thank you for this info

Reply:
I did some changes. But the article hasn't shown up in the `Basics` section
1. move article into the `docs`, and add `${id}` , `flatlist-on-viewable-item-changed` 
2. add `${id}` in `website/sidebars` 
3. fix the id in `versionedDocsBlacklist.json`. 
save changes, then, I run `yarn start` in the `react-native-website/website` , open `http://localhost:3000/`. But I can't see the article in the `docs-> The basics`

Reply:
@Simek would you shed some light on this?

Reply:
@sueLan You have only updated the `next`/`master` docs (https://reactnative.dev/versions), to see the changes in the current docs you need to edit `versioned_sidebar` and add the page to `versioned_docs` for the `0.63` release.

Reply:
Hmm, maybe when using `versionedDocsBlacklist` only the sidebar change is required. AFAIK blacklist prevents from generating static, versioned pages on new version cut. I'm not sure how it will behave in this situation.

If you will be still struggling with page visibility I will look on this PR locally during the weekend. 🙂

Reply:
ha Thank you ~, I did as you told me, and saw the page.  Besides, I updated a picture.